feature-libs/cart/base/core/facade/multi-cart.service.ts

```diff
@@ -85,7 +85,9 @@ export class MultiCartService implements MultiCartFacade {
    * Simple random temp cart id generator
    */
   protected generateTempCartId(): string {
-    const pseudoUuid = Math.random().toString(36).substring(2, 11);
+    const array = new Uint32Array(1);
+    window.crypto.getRandomValues(array);
```

Comment:
I would be careful with `window` as it could potentially break the SSR rendering process.

Reply:
The function is triggered when a new cart is created, which is unlikely to occur during the rendering process. However, it is recommended to check if the window object exists before using it.

Reply:
Reverted `globalThis.crypto` as it didn't work in SSR...